api_spec.yaml

```diff
@@ -100,7 +100,7 @@ paths:
                                   type: string
                                 lastName:
                                   type: string
-                                thirdparty:
+                                thirdParty:
```

Comment:
This changes to reflect the correct property naming based on the returned data from API